doc/hooks.rst

```diff
@@ -1,7 +1,7 @@
 Ganeti customisation using hooks
 ================================
 
-Documents Ganeti version 2.16
+Documents Ganeti version 3.0
```

Comment:
Do the checks really check that various (random?) design docs have been updated to the latest version?

Reply:
Yes, it seems so. See also [my mail on ganeti-devel](https://groups.google.com/forum/#!msg/ganeti-devel/Kj6i-YU9TCo/fVPZETdfAgAJ):
```bash
LC_ALL=C URL=file:///${HOME}/ganeti devel/release master
...
Incorrect version in doc/iallocator.rst, expected 3.0
Incorrect version in doc/hooks.rst, expected 3.0
Incorrect version in doc/virtual-cluster.rst, expected 3.0
Incorrect version in doc/security.rst, expected 3.0
File ../../doc/design-3.0.rst not found
doc/design-draft.rst was not updated for version 3.0
```
ATM it is beyond my comprehension, how much sense this makes. Or I'm doing something wrong? I wonder, why @apoikos didn't pick up this in #1469?

Reply:
I'm not sure I understand either why those specific docs are required to be the latest version. I should have, as I reviewed the original change (https://github.com/ganeti/ganeti/commit/7ecb3d0e0426293cf5522f30935e081a54c18676), but I don't remember at all. Well, LGTM then :)